sims/common-sim-flags.mk

```diff
@@ -21,3 +21,18 @@ SIM_LDFLAGS = \
 	-lfesvr \
 	-ldramsim \
 	$(EXTRA_SIM_LDFLAGS)
+
+SIM_FILE_REQS += \
+	$(TESTCHIP_RSRCS_DIR)/testchipip/csrc/SimSerial.cc \
+	$(TESTCHIP_RSRCS_DIR)/testchipip/csrc/SimDRAM.cc \
+	$(TESTCHIP_RSRCS_DIR)/testchipip/csrc/testchip_tsi.cc \
+	$(TESTCHIP_RSRCS_DIR)/testchipip/csrc/testchip_tsi.h \
+	$(TESTCHIP_RSRCS_DIR)/testchipip/csrc/mm.h \
+	$(TESTCHIP_RSRCS_DIR)/testchipip/csrc/mm.cc \
+	$(TESTCHIP_RSRCS_DIR)/testchipip/csrc/mm_dramsim2.h \
+	$(TESTCHIP_RSRCS_DIR)/testchipip/csrc/mm_dramsim2.cc \
+	$(ROCKETCHIP_RSRCS_DIR)/vsrc/EICG_wrapper.v \
```

Comment:
Why do we need to enumerate these here, instead of relying on addResource to copy these in.

Reply:
Hmm. So it looks like the `sim_files.common.f` which is generated from the BB files and `sim_files.f` holds the `.v/c/h` files altogether. So it looks like the only reason to have this is to add the non-blackbox files like `EICG_wrapper.v` and the VCS/Verilator specific files. I'll make adjustments tomorrow for this.

Reply:
Ok. `emulator.cc` requires the other `.cc` and `.h` files to compile for Verilator. VCS I don't think needs it.